composer.json

```diff
@@ -15,15 +15,15 @@
     ],
     "require": {
         "php": "^7.0",
-        "php-http/httplug": "^1.0",
+        "php-http/httplug": "^1.0 | ^2.0",
         "php-http/multipart-stream-builder": "^1.0",
-        "php-http/client-common": "^1.1",
+        "php-http/client-common": "^1.1 | ^2.0",
         "php-http/discovery": "^1.0"
     },
     "require-dev": {
         "phpunit/phpunit": "^5.7",
         "php-http/message-factory": "^1.0",
-        "php-http/curl-client": "^1.6",
+        "php-http/guzzle6-adapter": "^1.1 | ^2.0",
```

Comment:
There isn't any version of `curl-client` compatible with `httplug` 2.x, that's why I used `guzzle6-adapter` here.